spec/index.bs

```diff
@@ -1545,6 +1545,14 @@ interface GPUMapMode {
 
   **Returns:** {{ArrayBuffer}}
 
+  1. If |offset| is 0:
+
+    1. Set |offset| to |this|.{{[[mapping_range]]}}[0]
+
+  1. If |size| is 0 and |offset| is greater than or equal to |this|.{{[[mapping_range]]}}[0] and less than |this|.{{[[mapping_range]]}}[1]:
```

Comment:
this sounds like a validation clause, and it should be in `getMappedRange Valid Usage` rules as opposed to here